src/components/UserNicknamesPanel/UserNicknamesPanel.js

```diff
@@ -44,7 +54,9 @@ class UserNicknamesPanel extends React.Component {
   }
 
   _handleDeleteNickname = async (event) => {
-    await this.props.deleteNickname(event.target.name)
+    await this.props.deleteNickname(this.props.user, this.props.nicknames.find((nick) => {
```

Comment:
We should check the response for an error here and display accordingly.

use `isError()` from `flux-standard-action` to check the result from the action now.